README.rst

```diff
@@ -13,20 +13,48 @@ lcls_naming_tool
 
 A tool that checks the form and content of names with respect to the LCLS naming convention.
 
+Instructions
+------------
+
+To run the Python script:
+
+1. Set ``$ chmod 775 lcls_naming_tool.py`` to make the file executable.
+
+2. To check if a PV name is valid pipe in the PV name like so ``$ echo "MR2K4:KBO:PIP:01:PUMPSIZE" | ./lcls_naming_tool.py``
```

Comment:
We're probably going to need to build a more convenient entrypoint in the future, something that people can run as a single command from their path and get --help text on.

Reply:
I added the ability to find the app version with -v and --version using argparse. Will look into using argparse for the piping feature, a more convenient entry point, and --help.

Reply:
Can the code be approved without argparse? I can add it after the repo review in a PR. Alex mentioned that unit tests for the functions will need to be written before code can be approved. I'm working on those right now.

Reply:
I don't mind approving without these additional features- the goal for each PR is always forward progress without introducing bugs or maintenance issues.

Reply:
I agree with Zach here.  I'd argue we should be splitting out further additions into separate PRs.  This includes tests, cli (argparse), library-import-compatibility (for install into pcds-conda)

I suppose the goal of this review depends on what Alex expects from this initial PR review.  What does he consider to be "done" here?

Reply:
I think he basically wants some unit tests and everyone's approval that the code is ready for use in production (nothing is glaringly bad and breaking). I think the code is pretty much ready except for the unit tests (currently writing them right now).